packages/ui/src/views/pull-request/pull-request-list-page.tsx

```diff
@@ -0,0 +1,119 @@
+      //     <NoData
+      //       iconName="no-search-magnifying-glass"
+      //       title="No search results"
+      //       description={['Check your spelling and filter options,', 'or search for a different keyword.']}
+      //       primaryButton={{
+      //         label: 'Clear search',
+      //         onClick: noop
+      //       }}
+      //       secondaryButton={{
+      //         label: 'Clear filters',
+      //         onClick: noop
+      //       }}
+      //     />
+      //   )
+      // }
+      return (
+        // TODO add: no data state
+        <NoData
+          iconName="no-data-folder"
+          title="No pull requests yet"
+          description={['There are no pull requests in this project yet.', 'Create a new pull request.']}
+          primaryButton={{
+            label: 'Create pull request',
+            onClick: noop
+          }}
+        />
+      )
+    }
+    return (
+      <PullRequestList
+        handleResetQuery={noop}
+        // LinkComponent={LinkComponent}
+        pullRequests={pullRequests?.map((item: PullRequestType) => ({
+          author: item?.author,
+          name: item?.name,
+          // TODO: fix review required when its actually there
+          reviewRequired: !item?.is_draft,
+          merged: item?.merged,
+          comments: item?.comments,
+          number: item?.number,
+          is_draft: item?.is_draft,
+          // TODO: add label information to display associated labels for each pull request
+          // labels: item?.labels?.map((key: string, color: string) => ({ text: key, color: color })),
+          // TODO: fix 2 hours ago in timestamp
+          timestamp: item?.timestamp,
+          source_branch: item?.source_branch,
+          state: item?.state,
+          labels: item?.labels?.map((label: { text: string; color: string }) => ({
+            text: label?.text || '',
+            color: label?.color
+          }))
+        }))}
+        closed_prs={repoMetadata?.num_closed_pulls}
+        open_prs={repoMetadata?.num_open_pulls}
+      />
+    )
+  }
+  return (
+    <SandboxLayout.Main hasHeader hasSubHeader hasLeftPanel>
+      <SandboxLayout.Content>
+        <Spacer size={10} />
+        <Text size={5} weight={'medium'}>
+          Pull Requests
+        </Text>
+        <Spacer size={6} />
+        <div className="flex justify-between gap-5">
+          <div className="flex-1">
+            {/* TODO: add filter from pixelpoint */}
+            {/* <Filter sortOptions={SortOptions} /> */}
+          </div>
+          <Button variant="default" asChild>
+            <Link to={`/spaces/${spaceId}/repos/${repoId}/pull-requests/compare/`}>New pull request</Link>
+          </Button>
+        </div>
+        <Spacer size={5} />
+        {renderListContent()}
+        <Spacer size={8} />
+        <PaginationComponent
+          totalPages={totalPages}
+          currentPage={page}
+          goToPage={(pageNum: number) => setPage(pageNum)}
```

Comment:
Not an issue, but you can simplify here
```suggestion
          goToPage={setPage}
```